sdk/keyvault/azure-security-keyvault-certificates/src/private/certificate_serializers.hpp

```diff
@@ -92,4 +92,13 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     {
       static std::string Serialize(CertificateCreateParameters const& parameters);
     };
+
+    struct CertificateIssuerSerializer final
```

Comment:
We don't ever want to create instances of any of these classes on this file.  We should make them a `class` an delete its constructor. 

I think not all the deserializers are deleting its constructor, so, we can do the change to all of them together in another single PR.